### PR TITLE
Add scenarios in board acceptance

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: trello/features/acceptance/BoardAcceptance.feature [hunk 1]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+    When Sets a "POST" request to "/boards/BoardObject.id/labels"
+      | key   | value       |
+      | name  | nameOfLabel |
+      | color | yellow      |
+    And Sends request
+    Then Should return status code 200
+    And Validates response body
+    And Validates schema with "board_schema.json"
+    # Post condition
+    And Sets a "DELETE" request to "/boards/"
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Change this "/boards/" to  "/boards/BoardObject.id"

File: trello/features/acceptance/BoardAcceptance.feature [hunk 2]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+      | key | value            |
+      | id  | (BoardObject.id) |
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Delete this

File: trello/features/acceptance/BoardAcceptance.feature [hunk 3]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+    When Sets a "POST" request to "/boards/BoardObject.id/lists"
+      | key   | value                |
+      | name  | ListCreatedFromBoard |
+      | color | top                  |
+    And Sends request
+    Then Should return status code 200
+    And Validates response body
+    And Validates schema with "board_schema.json"
+    # Post condition
+    And Sets a "DELETE" request to "/boards/"
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Change this "/boards/" to "/boards/BoardObject.id"

File: trello/features/acceptance/BoardAcceptance.feature [hunk 4]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+      | key | value            |
+      | id  | (BoardObject.id) |
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Delete those lines

File: trello/features/acceptance/BoardAcceptance.feature [hunk 5]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+    When Sets a "POST" request to "/boards/OrganizationObject.id/idTags"
+      | key   | value                   |
+      | id    | (BoardObject.id)        |
+      | value | (OrganizationObject.id) |
+    And Sends request
+    Then Should return status code 200
+    And Validates response body
+    And Validates schema with "board_schema.json"
+    # Post condition
+    And Sets a "DELETE" request to "/boards/"
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Change this "/boards/" to "/boards/BoardObject.id"

File: trello/features/acceptance/BoardAcceptance.feature [hunk 6]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+      | key | value            |
+      | id  | (BoardObject.id) |
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Delete those lines

File: trello/features/acceptance/BoardAcceptance.feature [hunk 7]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+    And Saves response as "BoardObject"
+    When Sets a "POST" request to "/boards/BoardObject.id/powerUps"
+      | key   | value    |
+      | value | calendar |
+    And Sends request
+    Then Should return status code 200
+    And Validates response body
+    And Validates schema with "board_schema.json"
+    # Post condition
+    And Sets a "DELETE" request to "/boards/"
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Change this "/boards/" to "/boards/BoardObject.id"

File: trello/features/acceptance/BoardAcceptance.feature [hunk 8]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+      | key | value            |
+      | id  | (BoardObject.id) |
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Delete those lines

File: trello/features/acceptance/BoardAcceptance.feature [hunk 9]
```diff
@@ -59,3 +59,106 @@ Feature: Board
...
+    And Sends request
+    And Should return status code 200
```
Comment (oscarlopez8) [inline trello/features/acceptance/BoardAcceptance.feature]: Add a line at the end of file

**Review by oscarlopez8 — APPROVED** [review]

**Review by Amet015 — COMMENTED** [review]
please discomment all these

**Review by Amet015 — APPROVED** [review]

**Pull request merged**